src/overlays/actors/ovl_Bg_Hidan_Hamstep/z_bg_hidan_hamstep.c

```diff
@@ -102,8 +102,8 @@ s32 BgHidanHamstep_SpawnChildren(BgHidanHamstep* this, PlayState* play2) {
     s16 params;
     PlayState* play = play2;
 
-    pos = pos; // Required to match
     pos.y = this->dyna.actor.home.pos.y - 100.0f;
+    pos = pos; // Required to match
```

Comment:
why not :smile:

Reply:
This actually suppresses another warning but I forgot to mention it in the PR description

Reply:
why is it happy with pos=pos after just pos.y gets initialized... :shrug: